app/views/placements/support/schools/placements/index.html.erb

```diff
@@ -7,7 +7,7 @@
 
   <div class="govuk-grid-row">
     <div class="govuk-grid-column-two-thirds">
-      <h1 class="govuk-heading-l"><%= t(".placements") %></h1>
+      <h2 class="govuk-heading-m"><%= t(".placements") %></h2>
```

Comment:
This one was actually a second `h1` for the page it's displayed on, this matches other support headings for the index pages.

Reply:
Nice catch ⭐